munch-match/src/components/resultCard.module.css

```diff
@@ -0,0 +1,59 @@
+/* missing "button results" (line 9), "rating" (line 23) */
+
+.resultsHeader {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: flex-start;
+    text-align: left;
+    gap: 5px;
+}
+
+.actionContainer {
+    display: flex;
+    justify-content: space-between;
+    gap: 5px;
+    width: 100%;
+    align-items: center;
+}
+
+.actionButton {
+    background-color: white;
+    border: 2px solid black;
+    border-radius: 50px;
+    text-align: center;
+    flex: 1;
+}
+
+.infoContainer {
+    text-align: left;
+}
+
+.resultImageContainer {
+    display: flex;
+    /* flex-direction: flex-end; */
+}
+
+.resultImage {
+    width: 100%;
+    height: 300px;
+    object-fit: cover;
+    border-radius: 15px;
+}
+
+.buttonD {
```

Comment:
The naming here could be more descriptive. Just ".button" as a duplicate variable name could also work since we are specifying the module it is from.